frontend/README.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
```

Comment:
I would like to discuss the JavaScript code on Wednesday meeting, but we can merge it before we meet.